lib/std/child_process.zig

```diff
@@ -1375,3 +1379,49 @@ test "build and call child_process" {
     const ret_val = try child_proc.spawnAndWait();
     try testing.expectEqual(ret_val, .{ .Exited = 0 });
 }
+
+test "creating a child process with stdin/stdout behavior set to StdIo.Pipe" {
```

Comment:
nit: please dont write `stdin/stdout` as it is annoying to filter the test in shell on error